.teamcity/src/subprojects/build/core/NativeBuild.kt

```diff
@@ -32,6 +37,19 @@ class NativeBuild(private val entry: NativeEntry) : BuildType({
             }
             defineTCPPortRange()
         }
+
+        if (entry.os == OS.Windows) {
+            powerShell {
+                name = "Install Rust (rustup)"
+                scriptFile("install_rust_windows.ps1")
+            }
+        } else {
+            script {
+                name = "Install Rust (rustup)"
+                scriptFile("install_rust_unix.sh")
+            }
+        }
```

Comment:
Minor suggestion.
You could extract this code into an extension function and move into a separate file. Then you could do the same for `param(...)` and you would end up with two extensions `installRust` and `enableRustCompilation`. These extensions would be located in the same file and used from build configurations where it is necessary.